### PR TITLE
storage: add NVMeNamespaceGet

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Comment by glimchb** [comment]
tested like this
```
$ docker run --network=host --rm -it namely/grpc-cli call --json_input --json_output 127.0.0.1:50051 NVMeNamespaceCreate "{'namespace' : {'subsystem_id' : 'nqn.2016-06.io.spdk:cnode1' , 'bdev' : 'Malloc1' } }"
connecting to 127.0.0.1:50051
{}
Rpc succeeded with OK status
```
and then
```
$ docker run --network=storage_opi --rm namely/grpc-cli call --json_input --json_output opi-spdk-server:50051 NVMeNamespaceList "{'subsystem_id' : '1'}"
connecting to opi-spdk-server:50051
{
 "namespace": [
  {
   "name": "Malloc1"
  }
 ]
}
Rpc succeeded with OK status
 ```
and finally
```
$ docker run --network=storage_opi --rm namely/grpc-cli call --json_input --json_output opi-spdk-server:50051 NVMeNamespaceGet "{'subsystem_id' : '1', 'namespace_id' : '1'}"
connecting to opi-spdk-server:50051
{
 "namespace": {
  "name": "Malloc1"
 }
}
Rpc succeeded with OK status
```

File: storage/server/frontend.go [hunk 1]
```diff
@@ -205,8 +205,32 @@ func (s *server) NVMeNamespaceList(ctx context.Context, in *pb.NVMeNamespaceList
...
+					return &pb.NVMeNamespaceGetResponse{Namespace: &pb.NVMeNamespace{Name: r.Name}}, nil
+				}
+			}
+			msg := fmt.Sprintf("Could not find NSID: %d", in.GetNamespaceId())
+			log.Print(msg)
+			return nil, status.Errorf(codes.InvalidArgument, msg)
+		}
+	}
+	msg := fmt.Sprintf("Could not find NQN: %s", nqn)
+	log.Print(msg)
```
Comment (mestery) [inline storage/server/frontend.go:232]: Nit: You can just use `log.Printf()` and format the string directly if you want.

**Review by mestery — APPROVED** [review]
Nice work!

File: storage/server/frontend.go (hunk 1 above)
Comment (seroyer) [reply storage/server/frontend.go:232]: Except that it's returning the msg below, too.

Comment (glimchb) [reply storage/server/frontend.go:232]: exactly, had to return the err message

**Pull request merged**